LL/senchas_síl_ébir.trig

```diff
@@ -1258,7 +1261,7 @@
     a foaf:Person;
     irishRel:nomName "Bresal";
     rel:childOf <#Oengus>;
-    owl:sameAs <http://example.com/LL/eoganachta_casil.trig#Bresail>.
+    owl:sameAs <http://example.com/Rawl_B502/clann_faílbe_id_est_clann_donngaile.trig#Bressail>.
```

Comment:
Are you sure? All the other people in this line point to the Eoganachta.  This would be the only person in this line not in that file as well.

Reply:
Yup. I just checked and these are all instances of the pedigree of Cormac mac Cuillenáin. Clann Failbe (i.e. CLann Donngaile) seem to be a branch of the Eoganachta too. Remember that Rawl. B.502 makes far more use of section headings than the other MSS. Hence it looks like a completely different kin-group.

Reply:
Cool, go ahead and close any open conversations along those lines then as it was me just forgetting :)